mrp_sale_info/__manifest__.py

```diff
@@ -1,28 +1,28 @@
-# -*- coding: utf-8 -*-
-# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "MRP Sale Info",
     "summary": "Adds sale information to Manufacturing models",
-    "version": "9.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Manufacturing",
-    "website": "http://www.antiun.com",
+    "website": "https://github.com/oca/manufacture",
     "author": "Antiun Ingeniería S.L., "
```

Comment:
Please remove Antiun, OdooMRP team, Serv. Tecnol... and add Tecnativa

Reply:
Change done